src/Symfony/Component/Serializer/Tests/SerializerTest.php

```diff
@@ -81,7 +82,7 @@ public function testNormalizeNoMatch()
     {
         $this->expectException('Symfony\Component\Serializer\Exception\UnexpectedValueException');
         $serializer = new Serializer([$this->getMockBuilder('Symfony\Component\Serializer\Normalizer\CustomNormalizer')->getMock()]);
-        $serializer->normalize(new \stdClass(), 'xml');
```

Comment:
This change looks like a tiny BC break, on the other hand this test was relying on an (undocumented) edge case. WDYT @symfony/mergers?

Reply:
What’s the change? That previously you would get an exception with stdClass and now it would work? Or am I missing the point?

Reply:
I guess that test covers the case when we have no matching normalizer for the class that is normalizable. So its not really important which not-normalizable class we are trying to normalize

Reply:
I think what @dunglas means is that currently an instance of `stdClass` can be properly serialized if you register a custom normalizer that supports it, with this PR such normalizer wouldn't be called anymore. 
IMHO it's ok but only if we don't match classes extending `stdClass` (yes, one can do that..), I suggest using `\stdClass::class === \get_class($data)` instead of `instanceof`.

Reply:
@chalasr : the ObjectNormalizer has a [very low priority](https://github.com/symfony/symfony/blob/e3b513b5a154d0a0f5e97a27bfd8108084f0a0df/src/Symfony/Bundle/FrameworkBundle/Resources/config/serializer.xml#L71-L72) though. So I don't think that's an issue.

Reply:
Why not making it part of the object normalizer instead? It has a very low priority that would mitigate this issue.

Reply:
@ogizanagi thanks, good idea. Will check that tomorrow, I think that will work